index.html

```diff
@@ -37,6 +37,119 @@ <h2 class="font connect">LET'S CONNECT </h2>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li><button class="lang">html</button></li>
+        <li><button class="lang">css</button></li>
+        <li><button class="lang">javaScript</button></li>
+      </ul>
+      <button class="btn">See Project</button>
+    </div>
+    <div class="card">
+      <img class="work-img" src="images/Snapshoot2.png" alt="">
+      <h2>Multi-Post Stories</h2>
+      <ul>
+        <li class="canopy">CANOPY</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">Back End Dev</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li><button class="lang">html</button></li>
+        <li><button class="lang">css</button></li>
+        <li><button class="lang">javaScript</button></li>
+      </ul>
+      <button class="btn">See Project</button>
+    </div>
+    <div class="card">
+      <img class="work-img" src="images/Snapshoot3.png" alt="">
+      <h2>Tonic</h2>
+      <ul>
+        <li class="canopy">CANOPY</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">Back End Dev</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li><button class="lang">html</button></li>
+        <li><button class="lang">css</button></li>
+        <li><button class="lang">javaScript</button></li>
+      </ul>
+      <button class="btn">See Project</button>
+    </div>
+    <div class="card">
+      <img class="work-img" src="images/Snapshoot4.png" alt="">
+      <h2>Multi-Post Stories</h2>
+      <ul>
+        <li class="canopy">CANOPY</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">Back End Dev</li>
+        <li class="more"><img src="images/Counter.png" alt=""></li>
+        <li class="more">2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li><button class="lang">html</button></li>
+        <li><button class="lang">css</button></li>
+        <li><button class="lang">javaScript</button></li>
+      </ul>
+      <button class="btn">See Project</button>
+    </div>
+  </section>
+  <section class="about font align">
+    <h2 class="intro font">About Myself</h2>
+    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="font connect">LET'S CONNECT </p>
```

Comment:
- [ ] [Required] Please, Let's make sure to use the header tag `hx` on every sub-title such as this one.